.buildkite/pipeline.yml

```diff
@@ -19,10 +27,31 @@ steps:
   - wait: ~
```

Comment:
Why we are adding this wait step?

Reply:
This wait forces that the two steps defined previously (Run check static and Run unit tests) are finished before executing the following ones.

I wanted to force that way, so in case one of these two steps fail, integration tests are not run. Same behavior as in Jenkins.